### DISCUSSION TITLE
Incorrect operation of the script

### DISCUSSION THREAD

**Discussion by sakatass** [ilyhalight/voice-over-translation#243] [Q&A]
Good afternoon! After the update the translation stopped working. The default settings are "do not translate from my language", "reduce video volume to 15%" and translation from English to English. After trying to change the language on either side or adding/removing a check mark, it asks to restart the page, but after restarting the settings revert to the original ones. Please help me to fix this problem or please give me the code of the previous version.

**Comment by ilyhalight** [comment]
Is the error still occurring?

You can always find the code of previous versions by switching tags in the folder dist/vot.user.js or dist/vot-cloudflare.js:

![image](https://github.com/ilyhalight/voice-over-translation/assets/62353659/47588baf-f488-4499-8567-c1813ec18606)
